admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -372,3 +393,7 @@ You must run ``occ`` as your HTTP user; see
 
 When you are finished, take your ownCloud server out of 
 ``maintenance:singleuser`` mode.
+
+.. Links
+   
+.. _User Home Folder Naming Rule: https://doc.owncloud.com/server/9.1/admin_manual/configuration_user/user_auth_ldap.html#special-attributes
```

Comment:
Doing a quick review of the file, this can go as it's not referenced in the file. I'm assuming that it was at one point. That said, after doing some research version-specific links are achievable, but they're ugly. Have a look at [this discussion on StackOverflow](https://stackoverflow.com/questions/30920983/use-version-number-in-file-links-in-sphinx) as an example of what I mean. Gladly, it's not required.